lcmtypes/CMakeLists.txt

```diff
@@ -13,7 +13,6 @@ lcmtypes_build(LCM_TYPES
 			 lcmt_support_data.lcm
 			 lcmt_foot_flag.lcm
 			 lcmt_body_motion_data.lcm
-			 lcmt_atlas_command.lcm
```

Comment:
This type was duplicated from DRC into Drake, but the duplicate is no longer used or needed